src/api/app/views/webui2/webui/groups/index.html.haml

```diff
@@ -4,3 +4,28 @@
 .card.mb-3
   = render partial: 'webui/configuration/tabs'
   .card-body
+    %h3 Manage Groups
+    %table.responsive.table.table-sm.table-striped.table-bordered.w-100#manage-groups-table
+      %thead
+        %tr
+          %th Group Name
+          %th Members
+          %th Action
+      %tbody
+        - @groups.each do |group|
```

Comment:
N+1 queries, `includes(:users)`

Reply:
It's already included in the controller

Reply:
This is because the `order` that is done in the database level. I wonder, do we really want to order by name? I would let it by when it was added, so that it doesn't change the order after saving. If we want to keep this ordering, I would suggest to remove the `includes` from the controller, which saves us 2 database queries (this should only be done in the new UI, as in the old one the order is not there, but just moving it to the old view would make it)

Reply:
It's much better to order data alphabetically than by the creation date because it is constant. For example, the group 'Admins' will always be before 'OBS Developers'. Users would have to always use the search for finding a group because groups wouldn't be ordered. Nobody would remember after a few days when groups were created, so the order by creation date doesn't bring value at all.

Reply:
I fixed the N+1 queries by adding `order(:login)` on the `users` association for the Group model. I don't see why we wouldn't want to order the users by login, so I believe it is fine. Let me know what you think.